FA2sp/Miscs/Hooks.SaveMap.cpp

```diff
@@ -45,8 +57,17 @@ DEFINE_HOOK(428D97, CFinalSunDlg_SaveMap, 7)
             ss << "[";
             ss << section.first;
             ss << "]\n";
+            if (!std::strcmp(section.first, "Basic")) {
```

Comment:
It's really bad to have do this during each iteraing, move it ahead of this loop and just try pINI->WriteString and pINI->ReadString stuff, that will keep our processing effectively.